code/modules/client/client_procs.dm

```diff
@@ -156,6 +156,10 @@ var/next_external_rsc = 0
 		preload_rsc = external_rsc_urls[next_external_rsc]
 #endif
 
+	//wait here if the ticker is in the middle of setup
+	while(ticker.current_state == GAME_STATE_SETTING_UP)
+		stoplag()
```

Comment:
Jello was referring to joining the round, not joining the server.

How ever there is already protection against attempting to join the round while the ticker is setting up.

Ie, you can remove this commit.